packages/flutter_tools/lib/src/commands/fuchsia_reload.dart

```diff
@@ -421,10 +431,14 @@ class _PortForwarder {
       return new _PortForwarder._(null, 0, 0, null, null);
     }
     final List<String> command = <String>[
-        'ssh', '-F', sshConfig, '-nNT',
+        'ssh', '-F', sshConfig, '-nNT', '-vvv',
```

Comment:
Did you mean to invoke ssh with -vvv here and below, or is this just left over from testing?